common-functions

```diff
@@ -249,7 +254,22 @@ service_backup_deauth() {
   local SERVICE_ROOT="${PLUGIN_DATA_ROOT}/${SERVICE}"
   local SERVICE_BACKUP_ROOT="${SERVICE_ROOT}/backup/"
 
-  rm -rf "$SERVICE_BACKUP_ROOT"
+  rm -f "$SERVICE_BACKUP_ROOT/AWS_ACCESS_KEY_ID"
+  rm -f "$SERVICE_BACKUP_ROOT/AWS_SECRET_ACCESS_KEY"
+  rm -f "$SERVICE_BACKUP_ROOT/AWS_DEFAULT_REGION"
+  rm -f "$SERVICE_BACKUP_ROOT/AWS_SIGNATURE_VERSION"
+  rm -f "$SERVICE_BACKUP_ROOT/ENDPOINT_URL"
```

Comment:
Why this change?

Reply:
Because the auth command is no longer the only one using that directory, leaving it the way it was would also delete the S3_STORAGE_CLASS.